dotnetv3/ACM/README.md

```diff
@@ -0,0 +1,41 @@
+# ACM code examples for the AWS SDK for .NET v3
+
+## Overview
+
+This folder contains examples for using the AWS Certificate Manager (ACM) using the AWS SDK for .NET v3.
+
+The ACM creates private certificates for your internal resources and manage the certificate lifecycle centrally.
+
+## ⚠️ Important
+
+- Running this code might result in charges to your AWS account.
+- Running the tests might result in charges to your AWS account.
+- We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to
+  perform the task. For more information, see Grant least privilege.
+- This code is not tested in every AWS Region. For more information, see AWS Regional Services.
+
+## Code examples
+
+### Single actions
+
+- [Describing Certificates](DescribeCertificates/DescribeCertificate/)
+- [Listing Certificates](ListCertificates/ListCertificates/)
+
+## Running the examples
+
+After the example compiles, you can run it from the command line. To do so,
+navigating to the folder that contains the .csproj file, and run the following
+command:
+
+```
+dotnet run
+```
+
+Alternatively, you can run the example from within your IDE.
+
+## Additional resources
+
+- [AWS SDK for .NET Developer Guide](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/welcome.html)
+- [AWS SDK for .NET API Reference Guide](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/index.html)
```

Comment:
You should have three links here:
* Link to user/developer guide for ACM.
* Link to service API for ACM.
* Link to the section of the .NET SDK reference for ACM, not just the top level.